packages/debugger/src/service.ts

```diff
@@ -468,7 +470,22 @@ export class DebuggerService implements IDebugger, IDisposable {
 
     // Update the local model and finish kernel configuration.
     this._model.breakpoints.setBreakpoints(path, updatedBreakpoints);
-    await this.session.sendRequest('configurationDone', {});
+    // await this.session.sendRequest('configurationDone', {});
+  }
+
+  async pauseOnExceptions(): Promise<void> {
+    if (!this.session?.isStarted) {
+      return
+    }
+
+    console.log("Pause on exceptions");
+    const reply = await this.session.sendRequest(
+      'setExceptionBreakpoints',
+      {
+        filters: ["raise", "uncaught"],
```

Comment:
I think you have a typo here, it should be `raised` instead of `raise`.

Reply:
Hi @JohanMabille, indeed I missed this, however still no event is triggered when an exception happens :-|

Reply:
I'm gonna investigate this on the backend side today.

Reply:
Thanks! 🎊